### ISSUE TITLE
Rewrap fails when I have align* in a latex file.

### ISSUE THREAD

**Issue by karljs** [stkb/Rewrap#77]
When I have a LaTeX environment with a star in the name (which is very common) it seems to cause wrapping to break.

For example:

```latex
\begin{align}
\end{align}

long line of text long line of text long line of text long line of text long line of text
```

Wrapping the long line works fine.

If I change the environment to `align*` then wrapping the long line no longer works.

- Extension Name: rewrap
- Extension Version: 1.7.0
- OS Version: Darwin x64 17.4.0
- VSCode version: 1.20.1

**Comment by stkb** [comment]
Sorry about that, I goofed. Have just published a fix in v1.7.1.

Thanks for all the info!

**Comment by karljs** [comment]
Thanks for the quick fix!
